src/Symfony/Component/HttpClient/RetryableHttpClient.php

```diff
@@ -59,11 +62,18 @@ public function request(string $method, string $url, array $options = []): Respo
         $retryCount = 0;
         $content = '';
         $firstChunk = null;
+        $lastChunk = \microtime(true);
```

Comment:
this variable name is confusing. It does not contain a chunk (even more confusing, `$firstChunk` which looks related is indeed holding a Chunk)

Reply:
indeed. fixed